### PR TITLE
Updated IMU Constants

### PR DESCRIPTION
**Pull request by ShineBrightMeow**

### CODE REVIEW

**Review by ButterscotchV — APPROVED** [review]

**Comment by l0ud** [comment]
Afaik there is no MPU6050_SF constant

**Comment by ButterscotchV** [comment]
> Afaik there is no MPU6050_SF constant

It was added recently
https://github.com/SlimeVR/SlimeVR-Tracker-ESP/blob/main/src/consts.h#L64

File: server/core/src/main/java/dev/slimevr/tracking/trackers/udp/FirmwareConstants.kt [hunk 1]
```diff
@@ -12,6 +12,13 @@ enum class IMUType(val id: UInt) {
 	BMI160(8u),
 	ICM20948(9u),
 	ICM42688(10u),
+	BMI270(11u),
+	LSM6DS3TRC(12u),
+	LSM6DSV(13u),
+	LSM6DSO(14u),
+	LSM6DSR(15u),
+	MPU6050_SF(16u),
```
Comment (ButterscotchV) [inline server/core/src/main/java/dev/slimevr/tracking/trackers/udp/FirmwareConstants.kt:20]:
```suggestion
```

**Review by ButterscotchV — CHANGES_REQUESTED** [review]

File: server/core/src/main/java/dev/slimevr/tracking/trackers/udp/FirmwareConstants.kt (hunk 1 above)
Comment (ShineBrightMeow) [reply server/core/src/main/java/dev/slimevr/tracking/trackers/udp/FirmwareConstants.kt:20]: I was going to ask about this:

https://github.com/SlimeVR/slimevr-node/blob/b1fd86198406f9265191528940bf008ac9b6d68b/packages/firmware-protocol/src/constants.ts#L41

I will PR the firmware repository if MPU6050_SF needs to be added.

Comment (ImUrX) [reply server/core/src/main/java/dev/slimevr/tracking/trackers/udp/FirmwareConstants.kt:20]: yes it needs to

Comment (unlogisch04) [reply server/core/src/main/java/dev/slimevr/tracking/trackers/udp/FirmwareConstants.kt:20]: Im against it! It does not make sense to mix IMU with Fusion code!

Comment (unlogisch04) [reply server/core/src/main/java/dev/slimevr/tracking/trackers/udp/FirmwareConstants.kt:20]: see also my comment on:
https://github.com/SlimeVR/SlimeVR-Tracker-ESP/pull/322#discussion_r1544627331

**Comment by unlogisch04** [comment]
> > Afaik there is no MPU6050_SF constant
> 
> It was added recently https://github.com/SlimeVR/SlimeVR-Tracker-ESP/blob/main/src/consts.h#L64
This Constant is there to be able in Firmware to select the fusion.

To the Server class ImuID is responsible. 
https://github.com/SlimeVR/SlimeVR-Tracker-ESP/blob/main/src/consts.h#L28

File: server/core/src/main/java/dev/slimevr/tracking/trackers/udp/FirmwareConstants.kt (hunk 1 above)
Comment (ShineBrightMeow) [reply server/core/src/main/java/dev/slimevr/tracking/trackers/udp/FirmwareConstants.kt]: Updated PR. Removed MPU6050_SF.

Also, made PR for SlimeVR-Node: https://github.com/SlimeVR/slimevr-node/pull/41

**Review by ButterscotchV — APPROVED** [review]

**Review by ImUrX — APPROVED** [review]

**Comment by ImUrX** [comment]
please update the branch

**Comment by ShineBrightMeow** [comment]
> please update the branch

Branch updated

**Pull request merged**